examples/ConfigurationFromEnvironment.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+require __DIR__ . '/../vendor/autoload.php';
+
+use OpenTelemetry\SDK\Trace\TracerProviderFactory;
+
+//@see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md
+putenv('OTEL_RESOURCE_ATTRIBUTES=service.version=1.0.0');
+putenv('OTEL_SERVICE_NAME=example-app');
+putenv('OTEL_LOG_LEVEL=warning');
+putenv('OTEL_TRACES_SAMPLER=traceidratio');
+putenv('OTEL_TRACES_SAMPLER_ARG=0.95');
+putenv('OTEL_TRACES_EXPORTER=console');
+putenv('OTEL_TRACES_PROCESSOR=batch'); //TODO not in spec, @see https://github.com/open-telemetry/opentelemetry-specification/issues/2110
+putenv('OTEL_BSP_SCHEDULE_DELAY=10000');
+
+echo 'Creating Exporter From Environment' . PHP_EOL;
+
+$tracerProvider = (new TracerProviderFactory('example'))->create();
```

Comment:
Yes, definitely. I did have it catching those and setting null/noop as appropriate, but I thought that was too mysterious without having the capability to log errors and warnings. I will work on a follow-up PR to add logging, and then we can work towards not throwing any exceptions.
(Or, did you just mean the example would be better with a try/catch to illustrate that it can throw exceptions?)

Reply:
> (Or, did you just mean the example would be better with a try/catch to illustrate that it can throw exceptions?)

Yes, but if you want to work towards not throwing exceptions, it's not needed I guess.